IPython/utils/PyColorize.py

```diff
@@ -61,76 +61,70 @@
 
 # Build a few color schemes
 NoColor = ColorScheme(
-    'NoColor',{
-    'header'         : Colors.NoColor,
-    token.NUMBER     : Colors.NoColor,
-    token.OP         : Colors.NoColor,
-    token.STRING     : Colors.NoColor,
-    tokenize.COMMENT : Colors.NoColor,
```

Comment:
There are three changes of `tokenize.COMMENT` to `token.COMMENT`,  afterwards darker wants me to reformat these. I'm happy to revert the formatting if you feel it's not useful here.